apps/explorer/src/components/transactions/TxCardUtils.tsx

```diff
@@ -105,9 +97,13 @@ export const genTableDataFromTxData = (
                     />
                 </TxTableCol>
             ),
-            amounts: (
+            txns: (
                 <TxTableCol>
-                    <SuiAmount amount={suiAmount} />
+                    {transaction.transaction?.data.transaction.kind ===
+                    'ProgrammableTransaction'
+                        ? transaction.transaction.data.transaction.transactions
```

Comment:
`transaction.transaction.data.transaction.transactions` lol 🤔

Reply:
Haha yeah I realized how much we need to rename some of these APIs now...